pkg/router/template/router.go

```diff
@@ -253,10 +263,11 @@ func newTemplateRouter(cfg templateRouterCfg) (*templateRouter, error) {
 		captureHTTPResponseHeaders:    cfg.captureHTTPResponseHeaders,
 		captureHTTPCookie:             cfg.captureHTTPCookie,
 		httpHeaderNameCaseAdjustments: cfg.httpHeaderNameCaseAdjustments,
-
-		metricReload:        metricsReload,
-		metricReloadFailure: metricReloadFailure,
-		metricWriteConfig:   metricWriteConfig,
+		httpResponseHeaders:           cfg.httpResponseHeaders,
+		httpRequestHeaders:            cfg.httpRequestHeaders,
+		metricReload:                  metricsReload,
+		metricReloadFailure:           metricReloadFailure,
+		metricWriteConfig:             metricWriteConfig,
```

Comment:
Why did you delete the blank line?  If you keep the blank line, that keeps things logically grouped, and it makes the diff smaller:
```suggestion
		httpResponseHeaders:           cfg.httpResponseHeaders,
		httpRequestHeaders:            cfg.httpRequestHeaders,

		metricReload:        metricsReload,
		metricReloadFailure: metricReloadFailure,
		metricWriteConfig:   metricWriteConfig,
```

Reply:
ok